OpenTrons/opentrons-api
# feat(app): add device reset screen (clear pipette cal data and runsHistory)

# Overview
Currently only support clear pipette data and runsHistory since gripper isn't implemented yet.
stored data clear function and factory reset need a new endpoint clear data.
Those 3 things will be solved in a following PR.
This will close RCORE-493



# Test Plan
Check the data reset function works as expected.
1. RobotSettingsDashboard (from Navigation's `Settings`)
2. Tap Device Reset
3. Tap pipette, runsHistory or both of them
4. Tap clear button



# Changelog
- add Device Reset screen for pipette calibration data and runsHistory data


# Review requests



# Risk assessment
low
